packages/create-catalyst/src/index.ts

```diff
@@ -65,6 +65,12 @@ const createCommand = program
       .default(getPackageManager())
       .hideHelp(),
   )
+  .addOption(
+    new Option('--code-editor <editor>', 'Package manager to use for the project')
```

Comment:
```suggestion
    new Option('--code-editor <editor>', 'The code editor you plan to use to work on this project')
```

Open to rewording, though!

Reply:
Ah good catch!